app/dashboard/views.py

```diff
@@ -2966,6 +2966,13 @@ def get_profile_tab(request, profile, tab, prev_context):
                 'desc': 'Through PoH, upload a a video of yourself and get vouched for by a member of their community.',
                 'match_percent': 50,
                 'is_verified': profile.is_poh_verified
+            }, {
+                'ref': 'qd',
+                'name': 'Quadratic Diplomacy',
+                'icon_path': static('v2/images/project_logos/qd.svg'),
```

Comment:
@scco can i have a logo for the http://gitcoin.co/trust page that points to quadratic diplomacy

Reply:
@scco bump ^^

Reply:
using diplomacy logo after discussing with @scco